src/main/resources/db/changelog/v1.0/changelog-v1.0.yaml

```diff
@@ -0,0 +1,4 @@
+databaseChangeLog:
+  - include:
+      file: changes/2024_08_21_add_book_table.yaml
+      relativeToChangelogFile: true
```

Comment:
why do you need this file? why not add the changeset files directly to the db.changelog-master.yaml?

Reply:
To me, it is a format that is easier to read. So keep it in life=)